packages/assertions/src/expectations/isTrue.ts

```diff
@@ -0,0 +1,8 @@
+import { Answerable } from '@serenity-js/core';
+
+import { Expectation } from '../Expectation';
+
+export function isTrue(): Expectation<boolean> {
+  return Expectation.thatActualShould<boolean, boolean>(`have value that is`, true)
```

Comment:
I think if you used an alias instead we might get a nicer error message should the assertion fail:

```typescript
export function isTrue(): Expectation<boolean> {
    return Expectation.to(`be true`)
        .soThatActual(equals(true))
}
```

Reply:
Hi @jan-molak, 
I would love that short text as well, 

The error message - `be true` is put in front of does..

>  2) isTrue
>       contributes to a human-readable description:
>
>      AssertionError: expected '#actor ensures that true does be true' to equal '#actor ensures that true does have value that is true'
>      + expected - actual
>
>      -#actor ensures that true does be true
>      +#actor ensures that true does have value that is true

And the "does" comes from Ensure.toString() and Check.toString()

So I need to do a bigger rewrite if, so "does" is not a part of the picture.

Reply:
True, good point. Sorry, I was doing that review in the middle of the night so missed that :-)

Reply:
Just for later reference: 
The code I got working before I realized the larger changes where 

     export function isTrue() {
       return Expectation.to(`have value that is true`)
     .soThatActual(equals(true))
    }